cmd/agentbasedinstaller/client/main.go

```diff
@@ -272,7 +272,7 @@ func importCluster(ctx context.Context, log *log.Logger, bmInventory *client.Ass
 	}
 
 	clusterID := strfmt.UUID(ImportOptions.ClusterID)
-	_, err = agentbasedinstaller.ImportCluster(ctx, log, bmInventory, clusterID, ImportOptions.ClusterName, ImportOptions.ClusterAPIVIPDNSName, ImportOptions.ClusterConfigDir)
+	_, err = agentbasedinstaller.ImportCluster(os.DirFS(ImportOptions.ClusterConfigDir), ctx, log, bmInventory, clusterID, ImportOptions.ClusterName, ImportOptions.ClusterAPIVIPDNSName)
```

Comment:
Can we just keep it 
`agentbasedinstaller.ImportCluster(ctx, log, bmInventory, clusterID, ImportOptions.ClusterName, ImportOptions.ClusterAPIVIPDNSName, os.DirFS(ImportOptions.ClusterConfigDir)` instead?

Reply:
I just kept the consistency witht the previous [RegisterExtraManifests](https://github.com/openshift/assisted-service/blob/e9ec47f9b06eaf6efa285a5a8f2aebb1fdd42771/cmd/agentbasedinstaller/register.go#L182) signature